benchmarks/vectorized_channel_shuffle.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers import ChannelShuffle
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+
+
+class OldChannelShuffle(BaseImageAugmentationLayer):
```

Comment:
Unless the behavior is changing as part of this PR, we should add a PyTest to this benchmark to verify that the old/new implementations are numerically identical (like [this one](https://github.com/keras-team/keras-cv/blob/master/benchmarks/vectorized_random_translation.py#L259))

Reply:
@LukeWood to confirm

Reply:
Please see https://github.com/keras-team/keras-cv/pull/1433#issuecomment-1472992645